app/clusters/[clusterId]/page.tsx

```diff
@@ -305,68 +308,76 @@ export default async function ClusterDetailsPage({
           <div className="font-mono text-xl">killer-block proofs</div>
         </div>
         {/* // TODO: Replace with killer-block proofs data */}
-        {Array.from({ length: 6 }).map((_, i) => (
-          <div
-            key={i}
-            className="grid grid-cols-[1fr_repeat(4,_auto)] gap-x-6 border-b border-primary-border p-6"
-          >
-            <div className="col-start-1 row-span-2 grid grid-cols-1 grid-rows-subgrid">
-              <div className="font-mono text-lg text-primary">222566340</div>
-              <div className="font-sans text-xs text-body-secondary">
-                {formatTimeAgo(new Date(Date.now() - 1000 * 60 * 6))}
-              </div>
-            </div>
-            <div className="col-start-2 row-span-2 grid grid-cols-1 grid-rows-subgrid place-items-center">
-              <div className="font-mono text-sm">
-                proving: {prettyMs(204_000)}
-              </div>
-              <div className="font-sans text-xs text-body-secondary">
-                total to proof: {prettyMs(350_000)}
-              </div>
+        {killerBlockProofs.length === 0 ? (
+          <div className="py-16 text-center">
```

Comment:
<img width="1501" alt="image" src="https://github.com/user-attachments/assets/26f47a47-b084-4902-85c6-595d43ef1a52" />

`py-16` may be a bit much for the top spacing here... 4 rem away it feels like this comment is off on it's own, would halve it personally:

```suggestion
          <div className="py-8 text-center">
```

Reply:
agree :)